### ISSUE TITLE
GCC failure with pip, virtualenv - solved by installing python-devel

### ISSUE THREAD

**Issue by prefrontalvortex** [bmcfee/resampy#43]
**tl;dr - don't forget to install python-devel on a new system when trying to get resampy installed!**

Not a bug, just an oversight leading to confusion. Posting this for posterity, I'm sure someone else will run into this issue at some point. 

Librosa bought me here.[ My issue is pretty close to this issue](https://github.com/bmcfee/resampy/issues/25). 

System: Fedora 25, python 3.5.2, pip 9.0.1, virtualenv 15.1.0
gcc (GCC) 6.2.1 20160916 (Red Hat 6.2.1-2)

`pip install resampy` fails with 

```
creating build/temp.linux-x86_64-3.5/resampy
    gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/home/mike/ve/ml/lib/python3.5/site-packages/numpy/core/include -I/usr/include/python3.5m -c resampy/interp.c -o build/temp.linux-x86_64-3.5/resampy/interp.o
    resampy/interp.c:24:20: fatal error: Python.h: No such file or directory
     #include "Python.h"
                        ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```

I cloned the git repo and ran `python setup.py install` and got 

```
running build_ext
building 'resampy.interp' extension
gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/home/mike/ve/ml/lib/python3.5/site-packages/numpy/core/include -I/usr/include/python3.5m -c resampy/interp.c -o build/temp.linux-x86_64-3.5/resampy/interp.o
gcc: error: resampy/interp.c: No such file or directory
```

`pip install -e resampy` gives the identical error, `pip install .` likewise. 

And then I'm like, wait a minute, did I install python-devel? 
```
sudo dnf install python-devel
pip install resampy
```

*success* 

D'oh!

**Comment by prefrontalvortex** [comment]
Solved by installing python development package (`python-devel`)
